src/explorer.ts

```diff
@@ -264,8 +267,25 @@ class KubernetesResource implements KubernetesObject, ResourceNode {
         return kubefsUri(this.namespace, this.resourceId, outputFormat);
     }
 
-    getChildren(kubectl: Kubectl, host: Host): vscode.ProviderResult<KubernetesObject[]> {
-        return [];
+    async getChildren(kubectl: Kubectl, host: Host): Promise<KubernetesObject[]> {
+        if (this.kind !== kuberesources.allKinds.pod) {
+            return [];
+        }
+        const result = await kubectl.asJson<Pod>(`get pods ${this.metadata.name} -o json`);
+        if (result.succeeded) {
+            const pod = result.result;
+            let ready = 0;
+            pod.status.containerStatuses.forEach((status) => {
+                if (status.ready) {
+                    ready++;
+                }
+            });
+            return [
+                new DummyObject(`${pod.status.phase} (${ready}/${pod.status.containerStatuses.length})`),
+                new DummyObject(pod.status.podIP),
+            ];
+        }
+        return [ new DummyObject("Error") ];
```

Comment:
Pass `result.error[0]` to the `DummyObject` constructor as the optional `diagnostic` argument - this will surface the error as a tooltip.

Reply:
moved to a different PR.